pkgs/development/python-modules/bc-python-hcl2/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildPythonPackage, fetchPypi, nose }:
+
+let
+  lark-parser = buildPythonPackage rec {
+    pname = "lark-parser";
```

Comment:
Why did you not create a separate package for this?

Reply:
bc-python-hcl2 doesn't work with lark-parser from Nix Store. Should I add a `python-modules/lark-parser/0.7.8.nix` ?

Reply:
This or if the package is simple enough just overwrite the version and hash here.

Reply:
Anyway, add a comment explaining why the version is pinned.